packages/api/src/command/webhook/webhook.ts

```diff
@@ -176,14 +185,24 @@ export const sendPayload = async (
   payload: unknown,
   url: string,
   apiKey: string,
+  cxId: string,
+  timestamp: string,
   timeout = DEFAULT_TIMEOUT_SEND_PAYLOAD_MS
   //eslint-disable-next-line @typescript-eslint/no-explicit-any
 ): Promise<any> => {
   try {
+    const hmac = crypto
+      .createHmac("sha256", apiKey)
+      .update(cxId)
+      .update(JSON.stringify(payload))
+      .update(timestamp)
+      .digest("hex");
     const res = await axios.post(url, payload, {
       headers: {
         "x-webhook-key": apiKey,
         "user-agent": "Metriport API",
+        "X-Metriport-Signature": hmac,
```

Comment:
put hmac in header along with timestamp.

Reply:
lowercase everywhere else, lets be consistent and lower the case here too, please